Dockerfile

```diff
@@ -17,6 +17,7 @@ RUN if [ "$USE_CHINA_NPM_REGISTRY" = 1 ]; then \
   fi;
 
 RUN npm i -g npm
+RUN npm i
```

Comment:
CI changes are purposed here. See https://github.com/DIYgod/RSSHub/pull/8716

I would not suggest adding extra changes other than new route.

Reply:
Removed. BTW, I didn't understand want I should do for the checklist at 
PR creating step.
I think it's just for me to check I have done. Not all need to be done.
But later I think you mean I need to mark all as done for merging a PR?
If so I think it's better to give more hint for that contributor should mark all as done before the PR can be merged.

Reply:
> I think you mean I need to mark all as done for merging a PR?

No, you only need to tick what you have / what you know.

PR is merged manually by collaborators which are really busy.

Reply:
Ok, I see, thank you. BTW I added art engine in this PR. And will adding art engine in other PRs later.